x-pack/plugins/spaces/server/routes/api/v1/spaces.js

```diff
@@ -4,44 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import Boom from 'boom';
-import { omit } from 'lodash';
 import { routePreCheckLicense } from '../../../lib/route_pre_check_license';
 import { spaceSchema } from '../../../lib/space_schema';
 import { wrapError } from '../../../lib/errors';
-import { isReservedSpace } from '../../../../common/is_reserved_space';
 import { addSpaceIdToPath } from '../../../lib/spaces_url_parser';
 
 export function initSpacesApi(server) {
   const routePreCheckLicenseFn = routePreCheckLicense(server);
 
-  function convertSavedObjectToSpace(savedObject) {
-    return {
-      id: savedObject.id,
-      ...savedObject.attributes
-    };
-  }
-
   server.route({
     method: 'GET',
     path: '/api/spaces/v1/spaces',
     async handler(request, reply) {
-      const client = request.getSavedObjectsClient();
 
-      let spaces;
+      const spacesClient = server.plugins.spaces.spacesClient.getScopedClient(request);
```

Comment:
We'll need to sync this branch up with `spaces-phase-1` because this file doesn't exist anymore